includes/class-wc-calypso-bridge-woocommerce-admin-features.php

```diff
@@ -56,11 +58,29 @@ public function initialize() {
 		add_filter( 'woocommerce_admin_features', array( $this, 'filter_wc_admin_enabled_features' ) );
 		add_filter( 'woocommerce_admin_get_feature_config', array( $this, 'filter_woocommerce_admin_features' ), PHP_INT_MAX );
 
-		/**
+		/*
+		 * Hide the features under 'Advanced > Features' but let users disable our commerce-optimized menu.
+		 */
+		add_filter( 'woocommerce_get_settings_advanced', array( $this, 'filter_woocommerce_settings_features' ), 1000, 2 );
```

Comment:
As @xristos3490 [suggested](https://github.com/Automattic/wc-calypso-bridge/pull/1199/files#r1241987040) , I'll change this to PHPINTMAX priority